hubploy/__main__.py

```diff
@@ -52,6 +52,15 @@ def main():
     deploy_parser.add_argument(
         '--version',
     )
+    deploy_parser.add_argument(
+        '--timeout',
+        nargs=1
```

Comment:
nargs=1 produces a list with one item instead of the value itself. Is there a reason to want to do that?

Reply:
I did not know this. I will remove this.